docs/pt-br/gettingstarted.md

```diff
@@ -19,15 +19,15 @@ dependencies:
   bloc: ^3.0.0
 ```
 
-Para uma aplicação [Flutter](https://flutter.io), também é necessário adicionar o package flutter_bloc como uma dependência no `pubspec.yaml`.
+Para uma aplicação [Flutter](https://flutter.dev), também é necessário adicionar o package flutter_bloc como uma dependência no `pubspec.yaml`.
```

Comment:
```suggestion
Para uma aplicação [Flutter](https://flutter.dev/), também é necessário adicionar o package flutter_bloc como uma dependência no `pubspec.yaml`.
```

Reply:
why is the trailing slash necessary?

Reply:
It is not strictly necessary, but generally a trailing slash is used for the root path. Besides, it is nice to consistently stick to either having or not having a trailing slash (copying the URL will give you different results per browser).